packages/openapi-generator/src/file-serializer/schema.spec.ts

```diff
@@ -108,6 +108,15 @@ describe('serializeSchema for object schemas', () => {
 });
 
 describe('serializeSchema for array schemas', () => {
+  it('serializes array schema', () => {
```

Comment:
[req] Can you also add a test here for array of object type

Reply:
It is there just below this one.